cumulusci/tasks/bulkdata/tests/test_utils.py

```diff
@@ -46,6 +46,12 @@ def mock_sobject_describe(name: str):
             body=read_mock(name),
             status=200,
         )
+        responses.add(
+            method="GET",
+            url=f"https://example.com/services/data/v48.0/sobjects/{name.lower()}/describe",
+            body=read_mock(name),
+            status=200,
+        )
```

Comment:
```suggestion
```

This seems to be unnecessary

Reply:
I think it seems unnecessary on Mac but was added for Linux.

Reply:
But how does this end up lowercase on Linux? The describe call is made using the mapping's sf_object after _validate_sobject was called, so it should be canonicalized based on the objects from the global describe.